config.properties.example

```diff
@@ -125,6 +109,37 @@ kafka.metadata.fetch.timeout.ms=5000
 kafka.retries=0
 kafka.acks=1
 
+####### producer partitioning #######
+# used by kafka and kinesis
+
+# controls the input input into the hash function.  Note that this defines
+# which transactions keep ordering with respect to each other.  Generally
+# here you're making a trade-off between inter-row consistency and balanced
+# partitions.
+#producer_partition_by=database # [database, table, primary_key, column]
+
+# required when using producer_partition_by=column
+# otherwise the partitioner will revert to table
+# can be a single or multiple columns, e.g. aggregate_id or aggregate_id,event_type
+#producer_partition_columns=
+
+# required when using producer_partition_by=column
+# the fallback partitioning behavior when the specified column(s) do not exist
+# can be any one of [database, table, primary_key]
+#producer_partition_by_fallback=database
+
+######### kinesis ####################
+kinesis_stream=maxwell
+
+# AWS places a 256 unicode character limit on the max key length of a record
+# http://docs.aws.amazon.com/kinesis/latest/APIReference/API_PutRecord.html
+#
+# Setting this option to true enables hashing the key with the md5 algorithm
+# before we send it to kinesis so all the keys work within the key size limit.
+# Values: true, false
+# Default: false
+#kinesis_hash_key=true
```

Comment:
let's call this `kinesis_md5_keys`.  `kinesis_hash_key` made me think there as another something to configure before I read the text.